src/projectile.cpp

```diff
@@ -0,0 +1,221 @@
+#include "projectile.h"
+
+#include "game.h"
+#include "map.h"
+#include "map_iterator.h"
+#include "explosion.h"
+#include "field.h"
+
+projectile::projectile() :
```

Comment:
> Why?

Reduction in complexity

That said it's impossible to do so as set out in @Coolthulhu's argument